README.md

```diff
@@ -377,6 +377,89 @@ copycat.username('foo')
 #### `options`
 - **`limit`:** Constrain generated values to be less than or equal to `limit` number of chars
 
+### `copycat.unique(input, method, store, options)`
+
+The `unique` function is tailored to maintain uniqueness of values after they have undergone a specific transformation.
+This method is especially useful when the transformed values need to be unique, regardless of whether the input values are identical.
```

Comment:
Would it be worth explaining how it maintains this uniqueness (that it will keep calling `method` until it finds a unique value)